tests/unit_tests/training/test_checkpointing.py

```
@@ -1137,6 +1147,94 @@ def test_load_base_checkpoint_non_distributed_error(
+        mock_load_global,
+        base_config,
+        mock_pg_collection,
+    ):
+        """Direct iteration directory with torch_dist format delegates to _load_global_dist_base_checkpoint."""
+        mock_resolve.return_value = (_DIRECT_ITERATION_DIR_SENTINEL, False)
+        mock_get_format.return_value = "torch_dist"
+        mock_load_global.return_value = ({"model": "data"}, "/ckpt/iter_0001000", False, CheckpointType.GLOBAL)
+
+        result = _load_base_checkpoint("/ckpt/iter_0001000", base_config, rank0=True, pg_collection=mock_pg_collection)
+
+        state_dict, checkpoint_name, release, ckpt_type = result
```

Ruff (0.15.2)

[warning] Unpacked variable `checkpoint_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)